scripts/rest_client.py

```diff
@@ -156,6 +156,13 @@ def show_config(self):
         """
         return self._get("show_config")
 
+    def history(self, number=None):
+        """Return the amount of open trades.
```

Comment:
best change this line to `Return trades history."

also, i'd appreciate if you could add a line below (right before :return: - with `:param limit: Limit trades to latest X trades.`

Reply:
bad copy/paste :)
Done.